mediapipe/tasks/ios/genai/core/sources/LlmTaskRunner.swift

```diff
@@ -13,22 +13,21 @@
 // limitations under the License.
 
 import Foundation
-import LlmInferenceEngineC
+import MediaPipeTasksGenAIC
 
 /// This class is used to create and call appropriate methods on the C `LlmInferenceEngine_Session`
 /// to initialize, execute and terminate any MediaPipe `LlmInference` task.
 public final class LlmTaskRunner {
-  fileprivate typealias CLlmSession = UnsafeMutableRawPointer
```

Comment:
Was this change actually needed?

Reply:
Sorry this is a typo. I meant to make this
`private typealias CLlmSession = UnsafeMutableRawPointer`
private is the lowest access level where it will only be visible to the class declaration and extensions defined within same class. 
`fileprivate` means visible to anything declared in the same file even an entity outside the class.

I have fixed this in my new PR. You can choose to change it before merging.